extension/manifest.json

```diff
@@ -1,32 +1,40 @@
 {
   "manifest_version": 2,
-  "name": "speak-to-me",
+  "name": "Voice Fill",
   "version": "1.2.1",
+  "description": "Adds voice input to popular search pages in Firefox. Learn more about Voice Fill at https://testpilot.firefox.com",
   "developer": {
     "name": "Fabrice Desré",
```

Comment:
we should update this as well. prefer to use Firefox Test Pilot team or something along those lines, and identify the individuals on the experiment page ("brought to you by")